### PR TITLE
[Merged by Bors] - feat: port Algebra.Lie.BaseChange

### PR DESCRIPTION
**Pull request by Parcly-Taxel**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Lie/BaseChange.lean [hunk 1]
```diff
@@ -85,16 +85,17 @@ private theorem bracket_lie_self (x : A ⊗[R] L) : ⁅x, x⁆ = 0 := by
     · intro y₁ y₂ hy₁ hy₂
       simp only [add_add_add_comm, hy₁, hy₂, add_zero, LinearMap.add_apply, LinearMap.map_add]
 
-private theorem bracket_leibniz_lie (x y z : A ⊗[R] L) : ⁅x, ⁅y, z⁆⁆ = ⁅⁅x, y⁆, z⁆ + ⁅y, ⁅x, z⁆⁆ :=
-  by
+set_option maxHeartbeats 1500000 in
```
Comment (jcommelin) [inline Mathlib/Algebra/Lie/BaseChange.lean:88]: That's a lot of heartbeats for what seems to be a pretty simple statement + proof. Do we have any clue what's going on?

Comment (Parcly-Taxel) [reply Mathlib/Algebra/Lie/BaseChange.lean:88]: I don't know.

Comment (semorrison) [reply Mathlib/Algebra/Lie/BaseChange.lean:88]: Fixed by replacing some `simp`s by `rw`.

**Comment by semorrison** [comment]
bors d+

**Comment by Parcly-Taxel** [comment]
bors merge

**Pull request closed without merging**